runtime/port/common/j9sysinfo.c

```diff
@@ -194,3 +194,42 @@ IDATA
 j9sysinfo_get_cache_info(struct J9PortLibrary *portLibrary, struct const J9CacheInfoQuery * query) {
 	return J9PORT_ERROR_SYSINFO_NOT_SUPPORTED;
 }
+
+/**
+ * Initiates the J9CgroupMetricIteratorState attributes depending on subsystem
+ * @param[in] portLibrary The Port Library instance
+ * @param[in] subsystem It represents which Cgroup Subsystem state should be initiated
+ * @param[in] state Instance of the struct J9CgroupMetricIteratorState which gets initiated basing on the subsystem
+ * @return 0 on success error code on failure
+ */
+int32_t
+j9sysinfo_cgroup_subsystem_iterator_init(struct J9PortLibrary *portLibrary, uint64_t subsystem, struct J9CgroupMetricIteratorState *state)
+{
+	return J9PORT_ERROR_SYSINFO_CGROUP_SUBSYSTEM_UNAVAILABLE;
+}
```

Comment:
Why aren't these methods defined in OMR and used by OpenJ9?  We generally try and push the common implementation as low in the stack as possible

Reply:
@DanHeidinga Thank you for the review. I felt that these methods may be a good fit in OpenJ9 as its the only one which uses  these methods and that too only in the javadump use case. We could move it to OMR if any other future use cases for such mechanism arise. Let me know if this is ok with you.

Reply:
You're right that OpenJ9 is currently the only user.  I see this code as a really cool low level primitive that enables OpenJ9, or any runtime, to iterate over the defined cgroup systems.  Since OMR is the project that has the definitions of the systems, this code logically belongs in OMR.  

I would prefer to see the iterators moved to OMR so that other languages / runtimes can benefit from them as well.

Reply:
@DanHeidinga Thank you for your comments. I have created a [PR](https://github.com/eclipse/omr/pull/2994) basing on your review comments.

Please provide your views on it.

Thanks in advance